src/main/java/mate/academy/internetshop/service/ItemService.java

```diff
@@ -1,19 +1,18 @@
 package mate.academy.internetshop.service;
 
+import java.sql.SQLException;
 import java.util.List;
 import mate.academy.internetshop.model.Item;
 
 public interface ItemService {
 
-    Item create(Item item);
+    Item create(Item item) throws SQLException;
```

Comment:
```suggestion
    Item create(Item item);
```
we should move from interface to implementation, not from implementation to interface

Reply:
@boroda4436 I am sorry, but I don't clearly understand what did you mean by this remark. 
I have to add '''throws SQLException''' to both interface and implementation, or the project won't compile. 
Do You mean I should surround it with try/catch blocks in implementation?

Reply:
Давай так) Суть: ти спочатку пишеш Інтерфейс. І в ньому ти не декларуєш ніяких throws Exception, вірно? Отже твоя імплементація не повинна викидувати ніякі ексепшини. А в тебе виходить як: ти пишеш імплементацію,  ти викидуєш SQLException і в результаті ти повинен додати цю SQLException в інтерфейс щоб код компілювався. Так робити не треба. Уяви собі що ти імплементуєш інтерфейс, який був розроблений іншим розробником. Ти ж не можеш додати до нього свої ексепшини, вірно? Тому і тут ти не повинен рухатися від реалізації до інтерфейса, а навпаки - від інтерфейса до реалізації